doc/source/examples/multiphysics/sloshing-in-rectangular-tank/sloshing-in-rectangular-tank.rst

```diff
@@ -64,7 +68,7 @@ Parameter File
 Simulation Control
 ~~~~~~~~~~~~~~~~~~
 
-The results for this problem are highly sensitive to the accuracy of the time-stepping scheme. For this reason, we use a 2nd order backward differentiation scheme (``bdf2``) with a variable time step. The ``adaptive time step scaling`` is set to :math:`1.025` to ensure that the time-step does not rise too quickly during wave oscillations.
+The results for this problem are highly sensitive to the accuracy of the time stepping scheme. For this reason, we use a 2nd order backward differentiation scheme (``bdf2``) with a variable time step. The ``adaptive time step scaling`` is set to :math:`1.025` to ensure that the time step does not rise too quickly during wave oscillations.
```

Comment:
```suggestion
The results for this problem are highly sensitive to the accuracy of the time-stepping scheme. For this reason, we use a 2nd order backward differentiation scheme (``bdf2``) with a variable time step. The ``adaptive time step scaling`` is set to :math:`1.025` to ensure that the time-step does not rise too quickly during wave oscillations.
```

The correct form is "time-step" (with a hyphen) when used as a noun. The hyphen is used here to combine "time" and "step" into a single, unified term. Without the hyphen (as in "time step"), it might be unclear whether you're referring to the concept of a step in time or two separate ideas (a "time" and a "step").

Reply:
I've changed them!